pkg/proxy/ipvs/proxier.go

```diff
@@ -562,21 +563,21 @@ func newServiceInfo(port *v1.ServicePort, service *v1.Service, baseInfo *proxy.B
 	return info
 }
 
-// KernelHandler can handle the current installed kernel modules.
+// KernelHandler can handle the current installed kernel modules for ipvs.
 type KernelHandler interface {
+	kernel.Handler
 	GetModules() ([]string, error)
-	GetKernelVersion() (string, error)
 }
 
 // LinuxKernelHandler implements KernelHandler interface.
 type LinuxKernelHandler struct {
```

Comment:
I don't think we need this type anymore, we can call `NewLinuxKernelHandler()` directly?